src/engine/EventManager.cxx

```diff
@@ -244,15 +248,15 @@ void EventManager::checkEvents(SDL_Event &event, Engine &engine)
           {
             // Add highlighting here
           }
-          else if (!tileTypeEditMode.empty())
+          else if (!tileToPlace.empty())
```

Comment:
I think maybe we should rename it to   (currentTileID/TileIDToPlace/chosedTileID)
it would be more consistent with the rest of the code where we use tileID